### PR TITLE
Motion template fixes

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- Fix missing workflows in Motion create form.
- redesign motion meta box
- use table format for motion voting results

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Ok.

**Comment by emanuelschuetze** [comment]
@ostcar ping, please merge it today!

File: openslides/motions/static/templates/motions/motion-detail.html [hunk 1]
```diff
@@ -225,6 +232,7 @@ <h3 ng-if="motion.polls.length > 0" translate>Voting result</h3>
     </div>
     <div class="col-sm-1"></div>
   </div>
+  </div>
```
Comment (ostcar) [inline openslides/motions/static/templates/motions/motion-detail.html]: Please add indents so this two closing divs have different indents

**Comment by ostcar** [comment]
@emanuelschuetze Please fix the comment in another pull request

File: openslides/motions/static/templates/motions/motion-detail.html (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-detail.html]: done

**Pull request merged**